pkg/kritis/review/review_test.go

```diff
@@ -652,11 +652,20 @@ func TestGetAttestationAuthoritiesForISP(t *testing.T) {
 func makeRawAttestationPgp(signature, id string) metadata.RawAttestation {
 	return metadata.RawAttestation{
 		SignatureType: metadata.PgpSignatureType,
-		Signatures: []metadata.RawSignature{
-			{
-				Signature:   signature,
-				PublicKeyId: id,
-			},
+		Signature: metadata.RawSignature{
+			Signature:   signature,
+			PublicKeyId: id,
+		},
+	}
+}
+
+func makeRawAttestationGeneric(sig, id string, payload string) metadata.RawAttestation {
```

Comment:
Do you think it's better to combine this with `makeRawAttestationPgp` by adding `SignatureType` as another parameter?

Reply:
Done.